Practical 5 - Dictionaries, Code Review/state_names.py

```diff
@@ -0,0 +1,53 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+STATE_NAMES = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+               "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+# print(STATE_NAMES)
+
+for abbreviated_state, state in STATE_NAMES.items():
+    print("{:3} - {}".format(abbreviated_state, state))
+
+state = input("Enter short state: ").upper()
+while state != "":
+    if state in STATE_NAMES:
+        print(state, "is", STATE_NAMES[state])
+    else:
+        print("Invalid short state")
+    state = input("Enter short state: ").upper()
+
+# Color codes in a Dictionary
+COLOR_HEXS = {"AliceBlue": "#f0f8ff", "Cyan": "#00ffff", "DarkGreen": "#006400",
+              "Gold": "#ffd700", "Green": "#00ff00", "HotPink": "#ff69b4", "LightBlue": "#bfefff",
+              "Blue": "#0000cd", "Pink": "#ffc0cb", "Red": "#ff0000"}
+# User inputs color name and receives code
+color = input("Enter a color name: ")
+while color != "":
+    if color in COLOR_HEXS:
+        print("{} - {}".format(color, COLOR_HEXS[color]))
+    else:
+        print("Invalid color")
+    color = input("Enter a color name: ")
+
+word_dict = {}
+sentence = input("Text: ").lower()
+split_sentence = sentence.split()
+for word in split_sentence:
```

Comment:
See how this doesn't sound like "for singular in plural"? 
This implies one of them is not an ideal name. Each element IS a "word", so the list is a list of...?